code/game/objects/structures/crates_lockers/crates.dm

```diff
@@ -505,6 +505,30 @@
 	icon_opened = "electricalcrate_open"
 	icon_closed = "electricalcrate"
 
+/obj/structure/closet/crate/mail
+	name = "mail crate"
+	desc = "A plastic crate for mail delivery."
+	icon = 'icons/obj/bureaucracy.dmi'
+	icon_state = "mailsealed"
+	icon_opened = "mailopen"
+	icon_closed = "mailsealed"
+	material_drop = /obj/item/stack/sheet/plastic
+	material_drop_amount = 4
+	var/list/possible_contents = list(/obj/item/envelope/security,
+										/obj/item/envelope/science,
+										/obj/item/envelope/supply,
+										/obj/item/envelope/medical,
+										/obj/item/envelope/engineering,
+										/obj/item/envelope/service,
+										/obj/item/envelope/command,
+										/obj/item/envelope/misc)
```

Comment:
Indentation

Reply:
This will be updated in the next commit.